scripts/build-iso

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+cd $(dirname $0)/..
+
+echo "Start building ISO image"
+
+HARVESTER_INSTALLER_VERSION=master
```

Comment:
are we going to specify this version when doing a `tagged release`?

Reply:
This will ping to an installer tag when there's a release available. It is using the master head at the moment.